source/educators/how-tos/add_course_updates.rst

```diff
@@ -0,0 +1,6 @@
+Add Course Updates and Hand-outs
+###########################################
```

Comment:
Nit: These should be the same length as the above title. Don't need to go about fixing all, but don't add more.

Reply:
+1.  I figure I'll fix the ones I've touched, at least.